lecture/urls.py

```
@@ -0,0 +1,9 @@
+from django.urls import path
+from . import views 
+from django.conf.urls.static import static
+from django.conf import settings
+urlpatterns=[
+    path('home',views.lecture_home ,name="lecture_home"),
+    path('create_topic',views.create_title,name="create_topic"),
+    path('create_lecture',views.create_lecture,name="create_lecture")
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)